percy/driver_metadata.py

```diff
@@ -30,7 +30,10 @@ def capabilities(self):
     def session_capabilities(self):
         session_caps = Cache.get_cache(self.session_id, Cache.session_capabilities)
         if session_caps is None:
-            session_caps = dict(self.driver.desired_capabilities)
+            try:
+                session_caps = dict(self.driver.desired_capabilities)
+            except Exception:
+                session_caps = {}
```

Comment:
what all do we need from session caps ? if we are using nothing then why were we reading it in the first place ?

Reply:
actually we don't need session caps so removing it. anyways in current flow also we have spelling mistake due to which session caps are passed but not used in cli